tools/sgcollect_info

```diff
@@ -571,11 +577,35 @@ def main():
     if options.watch_stdin:
         setup_stdin_watcher()
 
-    # Get the sg url the user passed in, or use the default
     sg_url = options.sync_gateway_url
-    if sg_url is None:
-        sg_url = "http://127.0.0.1:4985"
-    print("Using Sync Gateway URL: {0}".format(sg_url))
+
+    if sg_url and "://" in sg_url:
+        pass
+    else:
+        if not sg_url:
+            root_url = "127.0.0.1:4985"
+        else:
+            root_url = sg_url
+        sg_url_tmp = "http://" + root_url
+        print("Trying Sync Gateway URL: {0}".format(sg_url_tmp))
+
+        try:
+            requests.get(sg_url_tmp)
```

Comment:
We're importing a new library in order to do this request. All others use `urllib2.urlopen`. Are we able to use that instead?

Reply:
No. The urlopen didn't try to communicate with the endpoint so much as it checked if it was alive. Ie. It'd say all is good but when you try to use http but you'd get a handshake error in SG.

Reply:
Are you sure? What about this use from further up (or everywhere else we get data out of SG over the REST API)?

The docs seem to suggest it'll perform a HTTP GET request, unless explicitly set to POST https://docs.python.org/2/library/urllib.html#urllib.urlopen 

```python
response = urllib2.urlopen(expvar_url(sg_url))
data = json.load(response)
```